src/main/java/life/qbic/exception/CustomException.java

```diff
@@ -0,0 +1,71 @@
+package life.qbic.exception;
+
+/**
+ * <p>An unspecific exception from the application.</p>
+ *
+ * @since 2.0.0
+ */
+public class CustomException extends RuntimeException {
```

Comment:
I think the name custom exception is not so fitting here. It's more like an ErrorCodeExpecption or so.
You basically provide the errorcode and the parameters of the error. Or maybe something like HTTPErrorException oder so